src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -396,6 +400,154 @@ describe("FlatTableRow", () => {
       });
     });
   });
+
+  describe("when the row is expandable", () => {
+    const SubRows = [
```

Comment:
Can you add keys to this array so there's not a console output when running the tests

Reply:
There's no console output about keys, I don't think they need them 👍🏻

Reply:
Weird I had one when I ran them but my virus scanner is causing npm to play up so it may have outputted from a different spec